### DISCUSSION TITLE
Sporadic errors in operation

### DISCUSSION THREAD

**Discussion by 011V32** [ronengr/hass_nuki_bt#51] [Q&A]
Hello.
I get errors every now and then.
Yesterday, for example, this one:

> Logger: pyNukiBT.nuki
> Source: runner.py:189
> First occurred: April 24, 2024 at 22:00:03 (2 occurrences)
> Last logged: April 24, 2024 at 22:00:03
> 
> HomeAssistant: Received unsolicited notification: Container: auth_id = b'\x04\x00\xf0\t' (total 4) command = (enum) CHALLENGE 4 payload = Container: nonce = b"x\x9d\xb1\xe8\x1f\x9b;A\x89'\x16\x89\x89\xca\xc8\x17"... (truncated, total 32) crc = 6179
> was expecting CHALLENGE

What could be the reason for this?
(The Bluetooth transceiver is approximately 1.5m away.)
Thanks.

**Comment by 011V32** [comment]
Hello @ronengr ,

maybe you could help?

> Logger: pyNukiBT.nuki
> Source: runner.py:190
> First occurred: May 14, 2024 at 22:00:05 (4 occurrences)
> Last logged: 07:30:10
> 
> HomeAssistant: Received unsolicited notification: Container: auth_id = b'\x04\x00\xf0\t' (total 4) command = (enum) CHALLENGE 4 payload = Container: nonce = b'@\x85g\xc5I\xb2=\xef8>\x03d\x1e>K\xb1'... (truncated, total 32) crc = 16558
> was expecting CHALLENGE
> HomeAssistant: Received unsolicited notification: Container: auth_id = b'\x04\x00\xf0\t' (total 4) command = (enum) CHALLENGE 4 payload = Container: nonce = b'3\xfe\x1e\xc8\xe7\xa4\x03O\x97i\xc01G\xdb~\x08'... (truncated, total 32) crc = 32513

Thx

**Comment by ronengr** [comment]
Are you using Nuki 4?
There is currently an issue when talking with nuki4, where getting the config info from nuki is not successful and results in some errors in the log.

**Reply by 011V32** [reply]
No. It's a Nuki 3.

**Comment by ronengr** [comment]
Could be a real communication issue (wireless interference etc)
although it is strange as from the logs it seems we are receiving what we where expecting to receive...

other then the error log line, are you experiencing any issues?

**Comment by 011V32** [comment]
With the version 0.0.14 I have no issues so far. 👍 
Thx.
